Message: Use the latest vue-eslint-parser

Tried to use the eslint-plugin-vue@beta, but i'm getting the following error: Use the latest vue-eslint-parser.

Here's my .eslintrc.js:

// http://eslint.org/docs/user-guide/configuring
module.exports = {
  root: true,
  parser: 'vue-eslint-parser',
  parserOptions: {
    sourceType: 'module',
    parser: 'babel-eslint',
        sourceType: 'module',
        allowImportExportEverywhere: false
  },
  env: {
    browser: true,
  },
  // https://github.com/feross/standard/blob/master/RULES.md#javascript-standard-style
  extends: ['eslint:recommended',
    'plugin:vue/recommended'],
  // required to lint *.vue files
  plugins: [   
    
  ],
  'settings': {
    
  },
  // add your custom rules here
  'rules': {
    // allow paren-less arrow functions
    'arrow-parens': 0,
    // allow async-await
    'generator-star-spacing': 0,
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    'camelcase': 2,
    'vue/no-invalid-template-root':2
  }
}


I installed the parser, and upgraded eslint to 4.0, but i'm still getting the error, both in vs code, and at compile time.
![image](https://user-images.githubusercontent.com/28810524/27248280-f066c5fe-52c6-11e7-829b-4a0002a6de45.png).

I have a related issue.

1. npm install -g eslint@3.19.0 eslint-plugin-vue@beta
2. <eslintrc.js>

`
module.exports = {
  extends: [
    'eslint:recommended',
    'plugin:vue/recommended'
  ],
  rules: {
    'vue/no-invalid-v-if': 'error',
    'vue/jsx-uses-vars': 2
  }
}
`

3. eslint src/components/dashboard.vue --ext .vue

And I get:

> Cannot find module 'vue-eslint-parser'
> Referenced from: plugin:vue/recommended
> Referenced from: /Users/edvin/Documents/Projects/sapi_app/.eslintrc.js
> Error: Cannot find module 'vue-eslint-parser'
> Referenced from: plugin:vue/recommended
> Referenced from: /Users/edvin/Documents/Projects/sapi_app/.eslintrc.js
>     at ModuleResolver.resolve (/usr/local/lib/node_modules/eslint/lib/util/module-resolver.js:74:19)
>     at load (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:554:42)
>     at configExtends.reduceRight.e (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:424:36)
>     at Array.reduceRight (native)
>     at applyExtends (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:408:28)
>     at load (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:566:22)
>     at configExtends.reduceRight.e (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:424:36)
>     at Array.reduceRight (native)
>     at applyExtends (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:408:28)
>     at Object.load (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:566:22)

--- a/lib/config/base.js
+++ b/lib/config/base.js
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
 
-  parser: 'vue-eslint-parser',
+  parser: require.resolve('vue-eslint-parser'),
 
   parserOptions: {
     ecmaVersion: 6,

